data/fetcher/fetcher.go

```
@@ -752,7 +752,11 @@ func (self *Fetcher) FetchOrderbook(timepoint uint64) {
 
 func (self *Fetcher) fetchPriceFromExchange(wg *sync.WaitGroup, exchange Exchange, data *ConcurrentAllPriceData, timepoint uint64) {
 	defer wg.Done()
-	exdata, err := exchange.FetchPriceData(timepoint)
+	fetchBTCPrice, err := self.globalStorage.GetBTCFetcherConfiguration()
+	if err != nil {
+		log.Printf("cannot get btc fetcher configuration")
```

the error is ignored.

---

still not fixed, the error should be handled somehow.

---

in the whole fetcher, go routine function are log only. Do you have any recommend for how to handle error here?